### DISCUSSION TITLE
Transactions

### DISCUSSION THREAD

**Discussion by recursivemoon** [philipmulcahy/azad#302] [General]
Hello - please consider adding transactions (including order ID) to the scrape options This would make it easier to reconcile with credit card.  Great work!

**Comment by philipmulcahy** [comment]
I believe (to the extent they are available) that this is already done, except that a couple of months ago Amazon slightly broke things and I've not put in the time to figure out how fixable it is: see #295

**Comment by philipmulcahy** [comment]
1.16.0 has basic transactions table support - the initial scrape is slow and doesn't give you any signal that things are happening until the table appears (unless you have the dev console open), but subsequent scrapes are fast (so long as you don't clear the cache).
